### PR TITLE
Look into parent for oval511 templates

### PR DESCRIPTION
**Pull request by mpreisler**
Sometimes the template is the same for OVAL 5.10 and 5.11. In other words sometimes we have a need to generate checks when using OVAL 5.11 that use nothing from OVAL 5.11 and can work with OVAL 5.10 features alone.

Unless this PR is merged you would have to copy the template into the 5.10 dir and the 5.11 dir. With this PR applied we will look into the OVAL 5.10 dir when generating from OVAL 5.11 templates when the template can't be found.

In other words we will look in these locations for the template:
- OVAL 5.11 dir
- OVAL 5.10 dir
- shared dir

In this order.

### CODE REVIEW

**Comment by mpreisler** [comment]
@openscap-jenkins test this please

**Comment by redhatrises** [comment]
Ack.
